docs/modules/install/pages/noseed.adoc

```diff
@@ -0,0 +1,47 @@
+= Install with no seed
+
+You may want to setup your decidim with no seeds.
+In this case, please follow the steps bellow:
+
+. Follow steps 1 through 5 of previous section(xref:install:manual.adoc[Manual Installation] section).
+
```

Comment:
I didn't try it, but as far as I know, if you remove the white lines then you can drop the "`[start=3]`" and also the hardcoded numbers (like the "3." and "4.")

Reply:
You are right about the numbering before the ordered list, but '[start=n]' seems to be essential in this case.

Reply:
I've just seen it in other changes that I was working on, you're absolutely right. How it is now it's perfect!